src/services/FormsService.ts

```diff
@@ -16,4 +17,14 @@ export class FormRPartAService extends ApiService {
   async getTraineeFormRPartA(): Promise<AxiosResponse<FormRPartA[]>> {
     return this.get<FormRPartA[]>(`/formr-parta/${this.traineeTisId}`);
   }
+
+  async saveTraineeFormRPartB(
+    formData: FormRPartB
```

Comment:
just for my own understanding, is there a reason why promise is being used over an observable?

Reply:
@omarCreativeDev  Would you need to bring in another library such as redux-observable to do this? Not sure what is gained by going down the rxjs route.

Reply:
@john12321 to be honest my react knowledge is quite limited but I assumed observables comes out of the box so im not entirely sure if another library would be needed : )

i do feel tho that observables provide a lot more out of the box but promises are of course also great